packages/client/hmi-client/src/stores/opened-workflow-node.ts

```diff
@@ -1,16 +1,28 @@
 import { defineStore } from 'pinia';
 import { ProjectAssetTypes } from '@/types/Project';
 
+interface StringValueMap {
+	[key: string]: number;
+}
+
 // Probably will be an array later
 export const useOpenedWorkflowNodeStore = defineStore('opened-workflow-node', {
 	state: () => ({
 		assetId: null as string | null,
-		pageType: null as ProjectAssetTypes | null
+		pageType: null as ProjectAssetTypes | null,
+		openedOutputIndex: 0 as number,
+		// model node
+		initialValues: null as StringValueMap[] | null,
+		parameterValues: null as StringValueMap[] | null
 	}),
 	actions: {
 		set(assetId: string | null, pageType: ProjectAssetTypes | null) {
```

Comment:
Nitpick; since there's more than one setter now, maybe we can rename this one to be more clear about what it is setting.